### ISSUE TITLE
Website 

### ISSUE THREAD

**Issue by Te-k** [Te-k/telegram-transparency#25]
TODO

**Comment by celsobessa** [comment]
Do you have any specific ideas, expectations or requirements for it?

**Comment by NoPlagiarism** [comment]
Fixed in https://github.com/Te-k/telegram-transparency/commit/8ee8087b1951572701a13fa98d1a41d4c4c38c05 ?

**Comment by Te-k** [comment]
Yes actually! But I am hoping to improve it and add a world map and a few graphs

**Comment by Roninon** [comment]
> Yes actually! But I am hoping to improve it and add a world map and a few graphs

@Te-k maybe I can help you with the web application?
Among my works there was a web map with more than hundred layers, lines and points. I used [leaflet.js](https://leafletjs.com/index.html) for displaying maps, as the main framework, I have experience working with vanilla js and [Vue.js](https://vuejs.org/)

As a reference, I can offer this option: [deepstatemap.live](https://deepstatemap.live)
